examples/rbac/README.md

```diff
@@ -0,0 +1,232 @@
+```yaml
+project: server
+registry:
+  registry_type: remote
+  path: feast-registry-server-feast-feature-server.feast-dev.svc.cluster.local:80
+offline_store:
+  type: remote
+  host: feast-offline-server-feast-feature-server.feast-dev.svc.cluster.local
+  port: 80
+online_store:
+  type: remote
+  path: http://feast-feature-server.feast-dev.svc.cluster.local:80
+auth:
+  type: kubernetes
+entity_key_serialization_version: 2
+
+```
+
+#### 5. **Apply User permissions and create Roles for examples(admin and user).**
+
+* As an example, we created two users: 1. admin and 2. user, and assigned the following roles and actions on the Feast object as shown in the below tables
+
+##### Roles and Permissions
+
+| **User**        | **Service Account** | **Roles**        | **Permission**          | **Feast Resources**                                                                                                                                      | **Actions**                                                                                                           |
+|-----------------|---------------------|------------------|-------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------|
+| admin           | feast-admin-sa      | feast-admin-role | feast_admin_permission  | FeatureView, OnDemandFeatureView, BatchFeatureView, StreamFeatureView, Entity, FeatureService, DataSource, ValidationReference, SavedDataset, Permission | CREATE, READ, UPDATE, DELETE, QUERY_ONLINE, QUERY_OFFLINE, WRITE_ONLINE, WRITE_OFFLINE                                |
+| user            | feast-user-sa       | feast-user-role  | feast_user_permission   | FeatureView, OnDemandFeatureView, BatchFeatureView, StreamFeatureView, Entity, FeatureService, DataSource, ValidationReference, SavedDataset, Permission | READ, QUERY_OFFLINE, QUERY_ONLINE                                                                                     |
+
+* For deploying client examples, the script will create a Deployment, roles, role bindings, and service accounts for each user. see roles configuration for admin [admin_resources.yaml](client%2Fadmin_resources.yaml) and user [user_resources.yaml](client%2Fuser_resources.yaml)
+* It will also create a ConfigMap on the client side that contains the test script and client-side feature store configuration.
+
+* To run the script below, if the server is already deployed, type 'no' for server deployment and 'yes' for client creation.
+
+   ```sh
+   ./install_feast.sh
+   ```
+
+* The applications should work without permission. 
+
+The Deployment of the overall setup looks like :
+
+![Deployment.png](deployment.png)
+
+![pod.png](pod.png)
+
+##### Apply Permissions
```

Comment:
I think this case can be fine too.
In the first step, the user can apply and verify that the application is working without setting up permissions. Then, the roles need to be created as part of the client setup, and permissions should be reapplied.

Reply:
There are no clear instructions on how to apply the store, and what folder to use, maybe when we create the Job it would make more sense. Closing for now

Reply:
agree that was not added clearly updated with more details and Sure, we will make the changes based on further enhancements.